<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncomePage</class>
 <widget class="QWidget" name="IncomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QPushButton" name="pbSalary">
      <property name="text">
       <string>Зарлата (подарки)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbSaved">
      <property name="text">
       <string>Забрать из сохранённых</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbCredit">
      <property name="text">
       <string>Взять из кредита/долг</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
